modules/twitch-notifications/events/botReady.js

```diff
@@ -0,0 +1,62 @@
+const {confDir} = require('../../../main');
+const config = require(`${confDir}/twitch-notifications/config.json`);
```

Comment:
This throws an error at first startup, as the config is not generated yet.

Ways to fix:
- Only load config when `botReady` fired, as this will ensure that configuration is loaded fully
- Wait for v2 as this will introduce `client.configurations`